skiko/src/jvmMain/cpp/common/StdVectorDecoder.cc

```diff
@@ -0,0 +1,20 @@
+#include <jni.h>
+#include "interop.hh"
+
+extern "C" JNIEXPORT jint JNICALL Java_org_jetbrains_skia_StdVectorDecoderKt_StdVectorDecoder_1nGetArraySize
+    (JNIEnv* env, jclass jclass, jlong ptr) {
+        std::vector<jlong>* vect = reinterpret_cast<std::vector<jlong> *>(ptr);
```

Comment:
Shalln't it be `std::vector<void*>` instead?

Reply:
Or even `std::vector<SkString>` to deal with string destruction